package.json

```diff
@@ -28,12 +28,16 @@
     "tape": "^4.0.0"
   },
   "dependencies": {
-    "config": "^1.14.0",
     "bluebird": "^2.4.2",
+    "config": "^1.14.0",
     "hapi": "^8.8.0",
     "joi": "^6.5.0",
     "json5": "^0.2.0",
     "primus": "^3.1.1",
+    "rootpath": "^0.1.2",
```

Comment:
hmm could you elaborate on this?
I don't exactly know where I should add spaces here...